modules/nw-sriov-dpdk-running-testpmd.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: REFERENCE
+[id="nw-sriov-dpdk-running-testpmd_{context}"]
+= Example testpmd script
+
+The following is an example script for running `testpmd`:
+
+[source,yaml]
```

Comment:
I just realised this should be "[source,terminal]".

Reply:
Done.